Dockerfile.dev

```diff
@@ -9,13 +9,19 @@ RUN apt-get update -y && \
       ln -s /usr/bin/clang-11 /usr/bin/clang && \
       ln -s /usr/bin/llc-11 /usr/bin/llc
 
+# Install Rust
+RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
```

Comment:
Should we pin a Rust version here?

Reply:
This is rustup so it's ok to install latest!